index.html

```diff
@@ -4,484 +4,528 @@
-                      <img src="./src/icons/avatar-6.svg" alt="second avatar" />
-                    </div>
-                    <div class="right-side__third-circle">
-                      <p class="avatar-text">5+</p>
-                      <img src="./src/icons/avatar-3.svg" alt="third avatar" />
-                    </div>
+                  <span class="card-content__company-name">Indeed</span>
+                </div>
+                <div class="card-content__avatars">
+                  <div class="right-side__first-circle">
+                    <img src="./src/icons/avatar-7.svg" alt="first avatar" />
+                  </div>
+                  <div class="right-side__second-circle">
+                    <img src="./src/icons/avatar-6.svg" alt="second avatar" />
+                  </div>
+                  <div class="right-side__third-circle">
+                    <p class="avatar_text">5+</p>
+                    <img src="./src/icons/avatar-3.svg" alt="third avatar" />
                   </div>
                 </div>
+              </div>
 
-                <!-- Cards content middle -->
-                <div class="cards-content__middle">
-                  <h2 class="cards-content__middle_title tltle">
-                    Project Manager
-                  </h2>
-                  <p class="cards-content__middle_subtitle">
-                    Full Time - Singapore
-                  </p>
-                </div>
+              <!-- card content middle -->
+              <div class="card-content__middle">
+                <p class="card-content__title">Project Manager</p>
+                <p class="card-content__subtitle">Full Time - Singapore</p>
+              </div>
 
-                <!-- Cards content bottom -->
-                <div class="cards-content__bottom">
-                  <a class="cards-content__button grey-btn" href="#">
-                    <p class="cards-content__button-text">Remote</p>
-                  </a>
-                  <a class="cards-content__button grey-btn" href="#">
-                    <p class="cards-content__button-text">Anywhere</p>
-                  </a>
-                  <p class="cards-content__button-text black-txt">
-                    $10k - $25k
-                  </p>
-                </div>
+              <!-- card content bottom -->
+              <div class="card-content__bottom">
+                <span class="card-content__feature" href="#"> Remote </span>
+                <span class="card-content__feature" href="#"> Anywhere </span>
+                <p class="feature__text">$10k - $25k</p>
               </div>
-            </div>
+            </a>
           </div>
 
           <!-- Scroll block -->
-          <a class="first-section__scroll-block" href="#">
-            <h2 class="scroll-block__text">View All</h2>
-            <img src="./src/icons/arrow.svg" alt="arrow" />
+          <a class="job-posts__scroll-block" href="#">
```

Comment:
```suggestion
          <a class="link_white" href="#">
```

This is definitely not a scroll block. This is a link. Create a universal class name for links and add modifiers if needed.